### ISSUE TITLE
Can we get a release of workflowcheck? ❤️ 

### ISSUE THREAD

**Issue by azdagron** [temporalio/sdk-go#1740]
Workflowcheck hasn't had a tagged release for ~ 9 months. Is there one planned soon?

Of particular interest, commit https://github.com/temporalio/sdk-go/commit/d10e87118a07b44fd09bf88d39a628f0e6e70c34 fixes handling of alias types introduced in go1.22. Would be great to have this available in an official release.

Thanks much!

**Comment by Quinn-With-Two-Ns** [comment]
Yes we plan to tag it soon after the next SDK release , that should be this week

**Comment by yuandrew** [comment]
Hi, we just tagged a new release for Workflowcheck! https://github.com/temporalio/sdk-go/releases/tag/contrib%2Ftools%2Fworkflowcheck%2Fv0.3.0
